frontend/components/notifications/NotificationsClientsTable.vue

```diff
@@ -0,0 +1,160 @@
+<script setup lang="ts">
+import { useNotificationsClientsStore } from '~/stores/notifications/useNotificationsClientsStore';
+import type { Cursor } from '~/types/datatable'
+
+defineEmits<{ (e: 'openDialog'): void }>()
+
+const { width } = useWindowSize()
+const cursor = ref<Cursor>()
+const pageSize = ref<number>(10)
+const { t: $t } = useTranslation()
+
+const {
+  clientsNotifications,
+  isLoading,
+  onSort,
+  query,
+  setCursor,
+  setPageSize,
+  setSearch,
+} = useNotificationsClientsStore()
+
+const colsVisible = computed(() => {
+  return {
+    footer: 1024,
+    threshold: width.value > 830,
+  }
+})
+
+const clientLink = computed(() => {
+  // TODO: implement client link when it's available from the API
+  // TODO: Test the endpoints
+  // return `/client/${data.clientId}`
+  return '/notifications'
+})
+</script>
+
+<template>
+  <div>
+    <BcTableControl
+      :title="$t('notifications.clients.title')"
+      :search-placeholder="$t('notifications.dashboards.search_placeholder')"
+      @set-search="setSearch"
+    >
+      <template #table>
+        <ClientOnly fallback-tag="span">
+          <BcTable
+            :data="clientsNotifications"
+            data-key="notification_id"
+            :cursor
+            :page-size
+            :selected-sort="query?.sort"
+            :loading="isLoading"
+            :add-spacer="true"
+            @set-cursor="setCursor"
+            @sort="onSort"
+            @set-page-size="setPageSize"
+          >
+            <Column
+              field="machine_name"
```

Comment:
could you check what the `field` prop is for and how it is supposed to be used

Reply:
> This property defines which key from the dataset will be displayed in the column. For instance, if you have a dataset where each item has properties like `timestamp`, `name`, or `age`, specifying field="timestamp" would display the corresponding value in the column. 

However, after removing the field prop in the `NotificationsClientsTable` component, the display remained unaffected. :eyes: